### DISCUSSION TITLE
Apache Pinot as Data source

### DISCUSSION THREAD

**Discussion by lalitbhagtani** [cuebook/CueObserve#195] [General]
Is support for apache pinot is planned for near future. We like to use cueobserve with our olap (apache pinot)

**Comment by sachinkbansal** [comment]
@lalitbhagtani We haven't yet planned for Pinot support. However, would be happy to consider it.

Have you already evaluated thirdeye? If yes, would love to understand why you are considering CueObserve over thirdeye.

**Reply by lalitbhagtani** [reply]
@sachinkbansal  one of our other service uses facebook prophet for forecasting, So to match results, our first choice is to use anomaly detection system which uses facebook prophet. Since cueobserve uses facebook prophet, we like to explore it first.
